odata-protocol/11.5 Operations.md

```diff
@@ -208,6 +208,10 @@ POST http://host/service/MyShoppingCart()/Items
 ```
 :::
 
+If the function returns a value of type `Edm.Stream` and no additional path
+segments follow the function invocation, the response to the `GET` request
+obeys the rules for [requesting stream properties](#RequestingStreamProperties).
+
```

Comment:
What happens if there are additional path segments after the function invocation? Do we allow anything other than an invocation of another function with binding parameter type Edm.Stream?

Reply:
I cannot imagine anything other than that. But what happens then is explained elsewhere (I hope).